components/resources/BUILD.gn

```diff
@@ -96,8 +97,15 @@ repack("resources") {
       sources += [ "$root_gen_dir/brave/components/brave_wallet/resources/brave_wallet.pak" ]
     }
 
-    deps += [ "//brave/components/brave_wallet/resources:ethereum_provider_generated_resources" ]
-    sources += [ "$root_gen_dir/brave/components/brave_wallet/resources/brave_wallet_script_generated.pak" ]
+    deps += [
+      "//brave/components/brave_wallet/resources:ethereum_provider_generated_resources",
+      "//brave/components/trezor_bridge/:copy_trezor_data",
```

Comment:
`copy_trezor_data` should be a dep internal to `brave/components/trezor_bridge/resources`

Reply:
the output path can be specified independently of the input path